src/cmd/policy.go

```diff
@@ -54,6 +64,26 @@ opslevel run policy -f policy.rego | jq
 	},
 }
 
+func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
+	if str, ok := a.Value.(ast.String); ok {
+		if _, err := os.Stat(string(str)); err != nil {
+			log.Warn().Msgf("%s", err)
+		} else {
+			file, err := os.Open(string(str))
+			cobra.CheckErr(err)
+			defer file.Close()
```

Comment:
do this before the cobra check error exits

Reply:
ah yeah, good catch